pyproject.toml

```diff
@@ -6,6 +6,16 @@ url = "https://github.com/isciences/exactextract"
 authors = [
   { "name" = "Daniel Baston" }
 ]
+requires-python = ">=3.9"
+classifiers = [
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+    "Operating System :: Microsoft :: Windows",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
```

Comment:
```suggestion
    "Programming Language :: Python :: 3.11",
    "Programming Language :: Python :: 3.12",
]
```

Is it important to list every supported version here? Every version for which binaries are distributed?

Reply:
This is not for binary but for the python package it self. I can help the users to understand which python version is supported or not

From pypi website: 

<img width="326" alt="Screenshot 2024-03-11 at 4 05 20 PM" src="https://github.com/isciences/exactextract/assets/10407788/483299d1-f6c0-4ff4-8a08-3146292f7535">

Reply:
If you want to keep this shorter, some packages also simply use the generic `"Programming Language :: Python :: 3"` to not have to constantly update this when there is a new Python version